### ISSUE TITLE
documentation on creating route to metadata service in Azure is unclear

### ISSUE THREAD

**Issue by boreal321** [F5Networks/f5-cloud-failover-extension#51]
## Description
The documentation here:

https://clouddocs.f5.com/products/extensions/f5-cloud-failover/latest/userguide/azure.html#using-tmsh

should be updated to clearly state that the gateway address in the example should be changed to the actual management interface gateway address on the BIG-IP that's been deployed. For example, after the code snipped we should add something like the following:

Please ensure that the gateway address  in the example above has been changed to the actual gateway you have configured on your BIG-IPs for the management interface. You can get that address by running the following `tmsh` command:

`tmsh list sys management-route`

## Environment information
- Cloud Failover Extension Version: all
- BIG-IP version: any
- Cloud provider: Azure

Severity: 4

**Comment by mikeoleary** [comment]
I have been confused by this also in the past when troubleshooting with customers. This was a silly oversight by me but still it could be clearer in the instructions.

I would recommend changing the instructions from:
````
tmsh create sys management-route metadata-route network 169.254.169.254/32 gateway 192.0.2.1
````
to something like this:
````
tmsh create sys management-route metadata-route network 169.254.169.254/32 gateway <your-mgmt-subnet-gw>
````

**Comment by nmenant** [comment]
Hi,

Thanks for the update. This is something we discussed internally a few days ago already and it's already being identified. Work in progress

**Comment by shyawnkarim** [comment]
Closing.

Updates to our Azure Quickstart guide were included with [Release 1.6](https://github.com/F5Networks/f5-cloud-failover-extension/releases/tag/v1.6.0).
